src/main/java/controller/LottoGameController.java

```diff
@@ -16,8 +15,11 @@ public LottoGameController(LottoGameView lottoGameView) {
 
     public void playGame() {
         Cost cost = requestCost();
-        Lottos lottos = generateLottos(cost);
-        displayLottos(lottos, cost);
+        int manualLottoAmount = lottoGameView.requestManualLottoAmount();
+        List<List<Integer>> manualLottoNumbers = lottoGameView.requestManualLottoNumbers(manualLottoAmount);
+
+        Lottos lottos = generateLottos(cost, manualLottoAmount, manualLottoNumbers);
+        displayLottos(lottos, cost, manualLottoAmount);
 
         WinningLotto winningLotto = requestWinningLotto();
         LottoResult lottoResult = calculateResult(lottos, winningLotto);
```

Comment:
이렇게 한 줄로 끝나는 메서드는 굳이 별도의 메서드로 분리하지 않는게 더 가독성이 좋은 것 같아요. 🙂

```suggestion
        LottoResult lottoResult = lottos.calculateResult(winningLotto);
```

Reply:
메서드를 그대로 이용하도록 변경했습니다. [c49d7e1](https://github.com/next-step/java-lotto-kakao/pull/36/commits/c49d7e1153b8923799dee34d2e92f61ac2899004)